libs/island-ui/core/src/lib/Tag/Tag.tsx

```diff
@@ -38,6 +39,7 @@ export const Tag = forwardRef<HTMLButtonElement & HTMLAnchorElement, TagProps>(
       disabled,
       label,
       bordered = false,
+      noBackgroundFill = false,
```

Comment:
I don't know about this abstraction. `variant = purple` sets background and text color to purple and `noBackgroundFill` then removes the background. It's dirty.

Variants clearly aren't doing it for `Tags` and we should let it accept a `background` and `color` which defaults to `transparent` and `blue400` or something.

Feel free to implement that but also feel free not to. This is something I could do later.

Reply:
I agree with all the comments and the more I thought about this I also don't think it's necessary to add tags prop to the LifeEventCard as I think we might just need to build a specific "Life Event Card in Category" later on because the UI of it is way different. I'm closing this PR and we'll create a more defined ticket for this later on 🚀